<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>word_training_kind</class>
 <widget class="QWidget" name="word_training_kind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择训练种类</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#word_training_kind{border-image: url(:/Yosemite-Color-Block.png);}</string>
  </property>
  <widget class="QPushButton" name="but_choose">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>单词选择训练</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_spell">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>单词拼写训练</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_back">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>23</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 84, 145);</string>
   </property>
   <property name="text">
    <string>选择训练种类:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
